java/src/test/java/org/certificatetransparency/ctlog/LogSignatureVerifierTest.java

```diff
@@ -56,13 +32,13 @@
 public class LogSignatureVerifierTest {
   /** Returns a LogSignatureVerifier for the test log with an EC key */
   private LogSignatureVerifier getVerifier() {
-    LogInfo logInfo = LogInfo.fromKeyFile(TEST_LOG_KEY);
+    LogInfo logInfo = LogInfo.fromKeyFile(LogSignatureVerifierTest.class.getResource(TEST_LOG_KEY).getFile());
```

Comment:
nit: Do `getClass().getResource(...)` instead of using the class name? The method is not static and `getClass()` tends to induce less copy-paste errors.

Same for others.

nit2: Please break lines at 100 columns. Eg:

```java
LogInfo logInfo =
    LogInfo.fromKeyFile(LogSignatureVerifierTest.class.getResource(TEST_LOG_KEY).getFile());
```

Google's line breaking rules are pretty particular, so your IDE autoformat is unlikely to be helpful here. (The rule is [break at a higher syntactic level](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break)).

In this particular case it'll go under 100 chars if you do `getClass()`, but please check the others as well.